stdlib/public/Concurrency/TaskLocal.swift

```diff
@@ -135,6 +136,9 @@ public final class TaskLocal<Value: Sendable>: Sendable, CustomStringConvertible
   /// If the value is a reference type, it will be retained for the duration of
   /// the operation closure.
   @discardableResult
+  @_unsafeInheritExecutor
+  @available(SwiftStdlib 5.1, *) // back deploy requires we declare the availability explicitly on this method
+  @_backDeploy(before: macOS 9999, iOS 9999, watchOS 9999, tvOS 9999) // SwiftStdlib 5.8 but it doesn't work with _backDeploy
```

Comment:
How sad. @tshortli can `@backDeploy` support `SwiftStdlib 5.8` here?

Reply:
`@_backDeploy` supports availability macros in general (they're heavily used in some of the tests). If there's an edge case exposed here, though, let me know @ktoso

Reply:
Yeah it didn’t work with the SwiftStdlib and I tried looking into it but didn’t quite see how those get applied. Help very welcome on this @tshortli !